sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/ParseDriver.scala

```diff
@@ -192,8 +192,17 @@ case object ParseErrorListener extends BaseErrorListener {
       charPositionInLine: Int,
       msg: String,
       e: RecognitionException): Unit = {
-    val position = Origin(Some(line), Some(charPositionInLine))
-    throw new ParseException(None, msg, position, position)
+    val (start, stop) = offendingSymbol match {
+      case token: CommonToken =>
+        val start = Origin (Some (line), Some (token.getCharPositionInLine) )
```

Comment:
You'll have to remove spaces around the parens for the style checker

Reply:
fixed. Not sure what happened there.